src/main/kotlin/io/snyk/plugin/ui/toolwindow/UIToolbox.kt

```diff
@@ -0,0 +1,49 @@
+package io.snyk.plugin.ui.toolwindow
+
+import com.intellij.uiDesigner.core.GridConstraints
+import com.intellij.uiDesigner.core.GridLayoutManager
+import java.awt.Container
+import java.awt.Dimension
+import java.awt.Insets
+import javax.swing.JPanel
+
+object UIToolbox {
```

Comment:
seems like we have UIUtils class already

Reply:
Oh yeah, didn't know, will move the methods there.